administrator/components/com_media/resources/scripts/components/toolbar/toolbar.vue

```diff
@@ -9,6 +9,7 @@
             </a>
         </div>
         <media-breadcrumb></media-breadcrumb>
+        <input type="text" :value="search" @input="changeSearch" placeholder="Search item.."/>
```

Comment:
You can use `v-model="search"` here and add a watcher to track changes of the `search `variable

Reply:
In watcher use a timeout to stop propagating mutation too much fast

Reply:
You can use the current code, no need to go for watchers as I and @dneukirchen has discussed, So please keep this way